test/modules/obsagent-client/docker/linux/amd64/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM ubuntu:18.04
```

Comment:
It looks like edgeAgent and edgeHub are using alpine as a base, should this do the same?

Reply:
Discussed with Noel and we should probably do the same.